packages/mdc-elevation/_mixins.scss

```diff
@@ -46,6 +47,113 @@
   }
 }
 
+///
+/// Called once per application to set up the global default elevation styles.
+///
+@mixin mdc-elevation-overlay-common($query: mdc-feature-all()) {
+  $feat-animation: mdc-feature-create-target($query, animation);
+  $feat-structure: mdc-feature-create-target($query, structure);
+
+  @include mdc-elevation-overlay-selector_ {
+    @include mdc-feature-targets($feat-structure) {
+      @include mdc-base-emit-once("mdc-elevation/common/structure") {
+        position: absolute;
+        top: 50%;
+        right: 50%;
+        bottom: 50%;
```

Comment:
Are right, bottom properties necessary?

Reply:
Likely not. GSS does some funky calculations with left/right swaps if only one property is specified. I'll remove for now and we can add them back if necessary.

Reply:
Ah yeah, I think you'll need the /* @noflip */ annotations above `left`, or the GSS compiler will flip it to `right` in RTL mode (which isn't necessary since we're just centering here).